Pipfile

```diff
@@ -19,7 +19,7 @@ requests = "*"
 fsspec = "==0.8.4"
 aiohttp = "==3.7.4"
 pyyaml = "==5.4.1"
-JSON-log-formatter = "==0.5.0"
+fybrik_python_logging = {path = "/home/cdoron/hello-world-python-package/dist/fybrik_python_logging-0.1-py3-none-any.whl"}
```

Comment:
This has your personal path

Reply:
This is temporary until we push the library to PYPI (or github).